tests/unit/amo/sagas/test_collections.js

```diff
@@ -34,6 +33,7 @@ import {
 describe(__filename, () => {
   const user = 'user-id-or-name';
   const slug = 'collection-slug';
+  const pageToFetch = 1;
```

Comment:
Could this just `page`? Then you could use the key/variable name shortcut below:

```js
.withArgs({
   api: state.api,
   page,
   slug,
   user,
})
```

Reply:
Good call. :-)